test/support/file_endpoint.ex

```diff
@@ -0,0 +1,84 @@
+defmodule Membrane.RTC.Engine.Support.FileEndpoint do
+  @moduledoc false
+
+  # Endpoint that publish data from file
+
+  use Membrane.Bin
+
+  alias Membrane.RTC.Engine
+  alias Membrane.H264
+  alias Membrane.FLV
+  require Membrane.Logger
+
+  @type encoding_t() :: String.t()
+
+  def_options rtc_engine: [
+                spec: pid(),
+                description: "Pid of parent Engine"
+              ],
+              file_path: [
+                spec: Path.t(),
+                description: "Path of file"
+              ],
+              track: [
+                spec: Engine.Track.t(),
+                description: "Track to publish"
+              ]
+
+  def_output_pad :output,
+    demand_unit: :buffers,
+    caps: {H264, stream_format: :byte_stream},
+    availability: :on_request
+
+  @impl true
+  def handle_init(opts) do
+    state = %{rtc_engine: opts.rtc_engine, file_path: opts.file_path, track: opts.track}
+
+    {:ok, state}
+  end
+
+  @impl true
+  def handle_prepared_to_playing(_ctx, state) do
+    {{:ok, notify: {:publish, {:new_tracks, [state.track]}}}, state}
+  end
+
+  @impl true
+  def handle_pad_added(Pad.ref(:output, {track_id, _rid}) = pad, _ctx, state) do
+    spec = %ParentSpec{
+      children: %{
+        source: %Membrane.File.Source{
+          location: state.file_path
+        },
+        demuxer: FLV.Demuxer,
```

Comment:
I am not sure about this one. I think we should get rid of it and read plain `.h264` file